language/en/interface.json

```
@@ -865,6 +865,9 @@
 			"transportmaxmass": "transport max mass",
 			"transportmaxsize": "transport max size",
 			"transportcapacity": "transport capacity",
+			"transport": "transport",
+			"light": "light",
+			"heavy": "heavy",
```

```suggestion
			"transportLight": "transport: light",
			"transportHeavy": "transport: heavy",
```
Entries in translation files should be complete phrases, rather than concatenating them in the widget, as languages have different subject-object-verb orders and punctuation meanings.